lib/render.test.js

```diff
@@ -183,18 +148,18 @@ describe(_.startCase(filename), function () {
       // THIS IS WHAT WE EXPECT FROM THE BASIC PROCESS:
 
       // 1. look up page
-      db.get.returns(resolveString({layout: '/components/hey'}));
+      db.get.returns(resolveString({layout: '/_components/hey'}));
 
       // 2. get the data that will be composed in the template
       components.get.returns(bluebird.resolve({}));
 
-      // 3. get the template that will be composed
-      components.getTemplate.returns('asdf');
+      // // 3. get the template that will be composed
+      // components.getTemplate.returns('asdf');
```

Comment:
should this (and other commented out lines from this PR) be removed from the test file? Hard to tell if a commented out part of a test is temporarily broken or purposeful removal

Reply:
Oh, good find. Forgot that was there.